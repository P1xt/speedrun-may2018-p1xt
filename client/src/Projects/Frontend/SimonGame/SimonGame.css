.SimonGame {
  text-align: left;
  margin-top: 40px; }
  .SimonGame > h1 {
    text-transform: none; }
