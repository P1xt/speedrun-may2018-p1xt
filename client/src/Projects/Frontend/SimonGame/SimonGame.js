import React from "react";
import { Container, Row, Col } from 'reactstrap';

import "./SimonGame.css";

const SimonGame = () => {
  return (
    <div className="SimonGame">
      <Container>
        <Row>
          <Col>
            <h1>Simon Game</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default SimonGame;
