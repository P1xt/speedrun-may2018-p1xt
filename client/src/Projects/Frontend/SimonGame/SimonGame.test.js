import React from "react";
import renderer from 'react-test-renderer';
import { shallow } from "enzyme";

import SimonGame from "./SimonGame";

describe("<SimonGame />", () => {
  it("renders without crashing", () => {
    shallow(<SimonGame />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<SimonGame />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
