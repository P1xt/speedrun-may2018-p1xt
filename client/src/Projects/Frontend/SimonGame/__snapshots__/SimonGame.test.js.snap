// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SimonGame /> renders correctly 1`] = `
<div
  className="SimonGame"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Simon Game
        </h1>
      </div>
    </div>
  </div>
</div>
`;
